src/connectors/notify/notify.connector.js

```diff
@@ -2,12 +2,24 @@ const { NotifyClient } = require("notifications-node-client");
 const { notifyClientDouble } = require("./notify.double");
 const { NOTIFY_KEY } = require("../../config");
 const { logEmitter } = require("../../services/logging.service");
+const {
+  updateNotificationOnCompleted
+} = require("../cacheDb/cacheDb.connector");
 
+/**
+ * Send a single email
+ * @param {string} templateId The template Id for the relevant email in notify
+ * @param {string} recipientEmail The email address for notifaction to be sent to
+ * @param {object} flattenedData The data for the template properties
+ * @param {object} pdfFile The pdf file to be sent with email
+ * @param {object} updateCache The function to update the  notification status
```

Comment:
It seems like you're always passing an object rather than a function

Reply:
removed now